docs/installation.md

```diff
@@ -0,0 +1,23 @@
+# Installation Guide
+
+## Requirement
+
+This SDK requires JDK 1.6+. We use [okhttp](https://github.com/square/okhttp) and [org.json](https://github.com/stleary/JSON-java) to manage project dependences.
+
+
+## Download
+
+Download [the latest JAR](https://github.com/yunify/qingstor-sdk-java/releases/download/2.1.1/qingstor-sdk-java-2.1.2.jar) or grab via Maven:
+
+```xml
+<dependency>
+  <groupId>com.yunify</groupId>
+	<artifactId>qingstor.sdk.java</artifactId>
+	<version>2.1.1</version>
```

Comment:
version changed.
Is there a better example for user? So that we don't need to maintain version number.